chart.js/index.d.ts

```diff
@@ -3,15 +3,15 @@
 // Definitions by: Alberto Nuti <https://github.com/anuti>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare enum ChartType {
+export enum ChartType {
```

Comment:
But even then `new window.Chart()` is not working, only `new window.Chart.Chart()`.
Is it better to wrap all in a namespace and export just the namespace and by default the Chart class?